console/src/components/Services/EventTrigger/Add/AddTrigger.js

```diff
@@ -443,7 +443,7 @@ class AddTrigger extends Component {
                 data-test="trigger-name"
                 placeholder="trigger_name"
                 required
-                pattern="^\w+$"
+                pattern="^[a-zA-Z0-9-_]+$"
```

Comment:
https://github.com/hasura/graphql-engine/blob/9af591e2cbb2f81dd7000908d230bd17eb1f5b8c/server/src-lib/Hasura/RQL/Types/Subscribe.hs#L126

Reply:
Can't start with _ or -

Reply:
Ideally should not end with - or _ as well